nixos/modules/services/misc/xmrig.nix

```diff
@@ -3,9 +3,16 @@
 
 let
   cfg = config.services.xmrig;
+  settings =
+    if cfg.cuda.enable then {
+      cuda.enabled = true;
+      cuda.loader = "${pkgs.xmrig-cuda}/lib/libxmrig-cuda.so";
+      cuda.nvml = "/run/opengl-driver/lib/libnvidia-ml.so";
```

Comment:
`"${addOpenGLRunpath.driverLink}/lib/libnvidia-ml.so"`, but also let's see if we can simplify this later